cypress.json

```diff
@@ -25,5 +25,6 @@
   "projectId": "ep9zr6",
   "viewportWidth": 1280,
   "viewportHeight": 768,
-  "chromeWebSecurity": false
+  "chromeWebSecurity": false,
+  "nodeVersion": "system"
```

Comment:
Is this needed for the test changes in the PR. Just wondering if it's a better fit for the upcoming cypress upgrade PR.

Reply:
For whatever reason I cannot run coverage reports without it.  I suppose we could pull it out, but it's a crippling issue.